programs/stake/src/stake_instruction.rs

```diff
@@ -4215,7 +4215,7 @@ mod tests {
         ] {
             // Set the source's starting balance to something large to ensure its post-split
             // balance meets all the requirements
-            let source_balance = u64::MAX;
+            let source_balance = 5_000_000;
```

Comment:
Thanks for finding these; we should definitely update these.

That being said, I'm currently trying to figure out why using 5M lamports (0.005 SOL) here doesn't fail for the case where the 1 SOL min stake delegation feature is enabled. Maybe < 1 SOL stake accounts can be created but the 1 SOL requirement is enforced on the delegate instruction ?